packages/fxa-auth-server/lib/payments/capability.ts

```diff
@@ -845,6 +888,36 @@ export class CapabilityService {
    * Retrieve the client capabilities
    */
   async getClients() {
+    const emptyClients: {
+      clientId: string;
+      capabilities: string[];
+    }[] = [];
+    const contentfulEnabled = config.getProperties().contentful.enabled;
+
+    if (contentfulEnabled) {
+      if (!this.capabilityManager) {
+        if (this.logToSentry('getClients.CapabilityManagerNotFound')) {
```

Comment:
If Contentful is enabled, we should be reporting every error, so `this.logToSentry` is no longer necessary.